src/algebra/ring/equiv.lean

```diff
@@ -248,44 +258,47 @@ end comm_semiring
 
 end opposite
 
-section non_unital_semiring
+section semiring
```

Comment:
Any reason you swapped the order of these sections? I think the previous order made more sense

Reply:
I think only because it used to be the case that the non-unital section was very short and came first, whereas the unital section was long, so I think I moved the unital one up so that it didn't move far, as it now only consists of `map_one`, `map_eq_one_iff` and `map_ne_one_iff`. If you'd like I can move it down after this section.

Reply:
I think it makes sense to deal with the weaker typeclasses first, so swapping them back would be a good idea

Reply:
(otherwise you can end up in a situation where you reprove the same result for the weak case after already proving it for the strong case, as the proof can only be reused in the other direction)